rust/src/composepost.rs

```diff
@@ -1201,7 +1201,18 @@ fn workaround_selinux_cross_labeling_recurse(
 /// This is the nearly the last code executed before we run `ostree commit`.
 pub fn compose_postprocess_final(rootfs_dfd: i32, _treefile: &Treefile) -> CxxResult<()> {
     let rootfs = unsafe { &crate::ffiutil::ffi_dirfd(rootfs_dfd)? };
-
+    if std::process::Command::new("add-determinism").status().expect("Failed to find add-determinism on system.").success() {
```

Comment:
This should probably be its own function like `run_add_determinisim(rootfs)`;

Reply:
Also this is not a great way to check for the existence of a binary. While the Rust stdlib doesn't have a single function for this I think it's trivial to do by combining

- `let path = std::env::var_os("PATH")`
- https://doc.rust-lang.org/std/env/fn.split_paths.html
-